src/02-room-details.js

```diff
@@ -49,9 +64,11 @@ function getRoomByDinosaurName(dinosaurs, rooms, dinosaurName) {}
       "Kit Hopkins Education Wing"
     ]
  */
-function getConnectedRoomNamesById(rooms, id) {}
+function getConnectedRoomNamesById ( rooms, id )
+{
+  rooms
```

Comment:
resubmit this and close this function.  This is preventing your test from running at all.